Revit_oM/Config/RevitPullConfig.cs

```diff
@@ -44,6 +44,9 @@ public class RevitPullConfig: ActionConfig
         [Description("If true, edges of elements will be pulled and stored under Revit_edges in CustomData.")]
         public virtual bool PullEdges { get; set; } = false;
 
+        [Description("If true, surfaces of elements will be pulled and stored under Revit_surfaces in CustomData.")]
```

Comment:
@adecler could you please guide us what would be the right way of versioning in such case? We are going to add a new property to the type + change its correspondent `Create` method.

Reply:
Oke, I've just tested it:
- created a component on master, saved the file
- compiled this branch and opened the file again:
    - the component did not have the new property input on canvas
    - it was processed correctly with the new property being equal to its default value (checked on debug)

Which means that the old scripts should work fine. 👍